### DISCUSSION TITLE
Watchman sonic Advance Leak detection

### DISCUSSION THREAD

**Discussion by CBDesignS** [merbanan/rtl_433#2907] [Q&A]
I am looking to find out where the leak detection could be hiding here.

{"model":"Oil-SonicAdv","id":5387924,"temperature_C":7,"depth_cm":71,"status":144,"mic":"CRC","protocol":"Watchman Sonic Advanced / Plus","rssi":-65,"duration":97000}

I know the obvious:-
id":5387924
temperature_C":7
oil depth_cm":71
rssi":-65

So could status":144 be the leak/rapid oil loss detector. If this is correct does anyone know what other values it changes to or could it be battery state.  This is a brand new sender so I only have data for about 1 week and this value has never changed

**Comment by zuckschwerdt** [comment]
Likely. But we didn't look for that field apparently. See https://github.com/merbanan/rtl_433/blob/master/src/devices/oil_watchman_advanced.c

Note that 144 is 0x90 and we know that 0x98 could be the normal state, just a single bit different.

See also https://github.com/merbanan/rtl_433/commits/master/src/devices/oil_watchman_advanced.c for links to changes and discussions.
